frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -0,0 +1,485 @@
+    )
+
+    switch (fetchResultWorkspaceDetail.apiResponse.status) {
+      case 200:
+        this.setState({
+          content: {
+            workspaceLabel: fetchResultWorkspaceDetail.body.label
+          }
+        })
+        break
+      default:
+        this.sendGlobalFlashMessage(props.t('Error while loading shared space detail'))
+    }
+  }
+
+  sendGlobalFlashMessage = msg => GLOBAL_dispatchEvent({
+    type: CUSTOM_EVENT.ADD_FLASH_MSG,
+    data: {
+      msg: msg,
+      type: 'warning',
+      delay: undefined
+    }
+  })
+
+  handleClickHideImageRaw = () => {
+    this.setState({ displayLightbox: false, fullscreen: false })
+  }
+
+  handleClickShowImageRaw = (fileSelected) => {
+    this.setState({ displayLightbox: true, fileSelected })
+  }
+
+  handleClickPreviousNextPage = previousNext => {
+    const { state } = this
+
+    let nextPageNumber = previousNext === DIRECTION.LEFT ? state.fileSelected - 1 : state.fileSelected + 1
+
+    if (previousNext === DIRECTION.RIGHT && state.fileSelected === state.imagesPreviews.length - 1) nextPageNumber = 0
+    if (previousNext === DIRECTION.LEFT && state.fileSelected === 0) nextPageNumber = state.imagesPreviews.length - 1
+
+    this.setState({
+      fileSelected: nextPageNumber
+    })
+  }
+
+  getPreviousImageUrl = () => {
+    const { state } = this
+
+    if (state.imagesPreviews.length === 1) return
+
+    if (state.fileSelected === 0) return state.imagesPreviews[state.imagesPreviews.length - 1].lightBoxUrlList[0]
+    return state.imagesPreviews[state.fileSelected - 1].lightBoxUrlList[0]
+  }
+
+  getNextImageUrl = () => {
+    const { state } = this
+
+    if (state.imagesPreviews.length === 1) return
+
+    if (state.fileSelected === state.imagesPreviews.length - 1) return state.imagesPreviews[0].lightBoxUrlList[0]
+    return state.imagesPreviews[state.fileSelected + 1].lightBoxUrlList[0]
+  }
+
+  onCarouselPositionChange = (fileSelected) => {
+    this.setState({ fileSelected })
+  }
+
+  handleOpenDeleteFilePopup = () => {
+    this.setState({
+      displayPopupDelete: true
+    })
+  }
+
+  handleCloseDeleteFilePopup = () => {
+    this.setState({
+      displayPopupDelete: false
+    })
+  }
+
+  deleteFile = async (filePosition) => {
+    const { state } = this
+    const contentIdToDelete = state.imagesPreviews[filePosition].contentId
+    const putResult = await handleFetchResult(
+      await putFileIsDeleted(state.config.apiUrl, state.config.appConfig.workspaceId, state.imagesPreviews[filePosition].contentId)
+    )
+    switch (putResult.status) {
+      case 204:
+        const newImagesPreviews = this.state.imagesPreviews.filter((image) => (image.contentId !== contentIdToDelete))
+        this.setState({
+          imagesPreviews: newImagesPreviews,
+          displayPopupDelete: false
+        })
+        break
+      default:
+    }
+  }
+
+  onSlickPlayClick (play) {
```

Comment:
naming convention is, if you start your function name with "on", it must be followed by an event type (like click, change, swipe, mouseover ...)
rename to onClickSlickPlay